src/pyOpenMS/tests/unittests/test_file_io.py

```
@@ -0,0 +1,453 @@
+def testIBSpectraFile():
+    """
+    @tests: IBSpectraFile
+     IBSpectraFile.__init__
+     IBSpectraFile.store
+    """
+    fh = pyopenms.IBSpectraFile()
+    cmap = pyopenms.ConsensusMap()
+    correctError = False
+    try:
+        fh.store(pyopenms.String("test.ibspectra.file"), cmap)
+        assert False
```

Ruff (0.14.10)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)